tulip_tables.html

```diff
@@ -284,9 +285,14 @@ <h4>Query Options</h4>
       style='margin-left:10px; vertical-align:top; width:auto'>
     <label for='node-input-includeDeleted' style='width:auto'>Include deleted tables</label>
   </div>
+  <div class='form-row' id='div-node-input-includeRequestInResult'>
```

Comment:
nit: can we move this up to the main section (ie, just above Query Type) since it no longer only relates to the query params, and copy the formatting from the HTTP keep-alive checkbox for consistency?